tests/org.codechecker.eclipse.rcp.unit.tests/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was needed to specify the aforementioned  JUnit Plug-In test  test running environment.